src/messageComposer/messageComposer.ts

```diff
@@ -464,7 +484,7 @@ export class MessageComposer extends WithSubscriptions {
       const draft = event.draft as DraftResponse;
       if (
         !draft ||
-        !!draft.parent_id !== !!this.threadId ||
+        (draft.parent_id ?? null) !== (this.threadId ?? null) ||
```

Comment:
Why this change?

Reply:
This fixes point number 5. - in `ThreadList`, if you have 2 or more `thread`s that belong to the same channel and you update the draft of one of them all of them would update with the same value (because `!!draft.parent_id !== !!this.threadId` is `false` for something like `draft.parent_id === '123'` and `this.threadId === '321'`.